### PR TITLE
[CHI-272] Renamed text color utility from inverse to light

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

File: src/chi/components/avatars/avatars.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@ $avatar-sizes: (
   lg: 4rem,
   xl: 6rem,
   xxl: 8rem);
-$avatar-colors: (grey, red, pink, purple, indigo, navy, blue, teal, mint, green, yellow, orange, inverse, secondary, primary);
+$avatar-colors: (grey, red, pink, purple, indigo, navy, blue, teal, mint, green, yellow, orange, light, secondary, primary);
```
Comment (jllr) [inline src/chi/components/avatars/avatars.scss]: @davidojedacl we shouldn't remove the inverse color from avatars, it should stay as legacy code, the same that we did with buttons.

File: src/website/views/components/avatar.pug [hunk 2]
```diff
@@ -156,27 +156,27 @@ ul#a-tabs--colors.a-tabs
         <i class="a-icon icon-user"></i>
       </div>
 
-h4 Inverse and transparent background
+h4 Light and transparent background
 p.-text
-  | Both icon and initial avatars support inverse version and transparent background.
+  | Both icon and initial avatars support light version and transparent background for dark backgrounds.
```
Comment (jllr) [inline src/website/views/components/avatar.pug]: @davidojedacl can you change from "light version and transparent background for dark backgrounds" to "light and transparent versions for dark backgrounds."?

File: src/website/views/components/icon.pug [hunk 3]
```diff
@@ -95,36 +95,54 @@ p.-text Two alternate small classes are available for specific app scenarios. Us
...
-        div.-p--2
-          div(class=`a-icon -md -text--${color}`)
+  .a-grid.-noGutter
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap
+      each type in ['text--body', 'text--primary', 'text--success', 'text--warning', 'text--danger', 'text--muted']
+        div.-p--1.-p-sm--2.-text--center
+          div(class=`a-icon -md -${type}`)
             svg
               use(xlink:href='#icon-logo-centurylink')
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap(style="background:#000;")
```
Comment (jllr) [inline src/website/views/components/icon.pug]: @davidojedacl can you delete (style="background:#000;") and add -bg--black class?

File: src/website/views/components/spinner.pug [hunk 4]
```diff
@@ -198,16 +198,31 @@ p.-text
...
       each type in ['text--primary', 'text--success', 'text--warning', 'text--danger', 'text--muted']
-        div.-p--2
+        div.-p--2.-text--center
           svg(class=`a-spinner -${type} -md -m--3`,viewBox=`0 0 66 66`)
             title Loading
             circle(class=`path`,cx=`33`,cy=`33`,r=`30`,fill=`none`,stroke-width=`6`)
-            p.-text(class=`-${type}`)
+            p.-text(class=`-${type} -px--1`)
               = `-${type}`
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap(style="background:#000;")
```
Comment (jllr) [inline src/website/views/components/spinner.pug]: @davidojedacl can you delete (style="background:#000;") and add -bg--black class?

**Comment by jllr** [comment]
@davidojedacl I think it can be confusing to continue referencing to -text--inverse in Spinner > Backdrop > Base. I would change to -text--white, as we are doing it in the rest of components. @mattnickles thoughts?

Screen Shot 2019-05-30 at 3 43 49 PM

File: src/chi/components/avatars/avatars.scss (hunk 1 above)
Comment (davidojedacl) [reply src/chi/components/avatars/avatars.scss]: done

File: src/website/views/components/avatar.pug (hunk 2 above)
Comment (davidojedacl) [reply src/website/views/components/avatar.pug]: done

File: src/website/views/components/icon.pug (hunk 3 above)
Comment (davidojedacl) [reply src/website/views/components/icon.pug]: done

File: src/website/views/components/spinner.pug (hunk 4 above)
Comment (davidojedacl) [reply src/website/views/components/spinner.pug]: done

**Comment by jllr** [comment]
@davidojedacl also noticed Cards > Hero is having .-text--inverse can you change to -text--light?

**Pull request merged**